large_media

```diff
@@ -1 +1 @@
-Subproject commit 547732eca85f14cf545ebc86875535afc38121c5
+Subproject commit 19f9bb9a092270d4a317db0c6d21a7a495af1e27
```

Comment:
**Caution!** This contains a submodule update

Reply:
large_media PR for images used in ESM markdowns